Makefile

```diff
@@ -35,11 +35,15 @@ format: ## Format Rego rules
 	@opa fmt -w lib/ validator/
 
 .PHONY: build
-build: format build_templates ## Format and build
+build: format build_templates check_sample_files ## Format, build and verify each template has a sample associated 
```

Comment:
I don't think this should necessarily be included in a "build" step.

Reply:
updated the build target back to: format build_templates